src/aura/HH_Canvas/HH_Canvas.cmp

```diff
@@ -17,9 +17,9 @@
 
     <!-- markup follows... -->
 
-    <ul id="SortCanvas" class="slds-list--horizontal slds-has-cards--space" style="display: inline;">
+    <ul id="SortCanvas" class="slds-list--horizontal slds-has-dividers--around-space slds-wrap">
```

Comment:
did you test this in the manage household ui to make sure drag/drop of the cards still works.  I only put in styles when I had no choice to get something working.